index.js

```diff
@@ -177,6 +178,16 @@ function start(clusterProcessId = 0) { // eslint-disable-line max-statements
     .readFileSync(conf.api.swagger.doc, ENCODING);
   const swaggerDoc = yaml.safeLoad(swaggerFile);
 
+  if (featureToggles.isFeatureEnabled('hideRoutes')) {
+    for (let _path in swaggerDoc.paths) {
+      if (swaggerDoc.paths.hasOwnProperty(_path)) {
+        if (conf.hiddenRoutes.includes(_path.split('/')[ONE])) {
```

Comment:
@BrandonMurnane is conf.hiddenRoutes defined somewhere or does this resolve to an environment variable? I don't see is in this PR

Reply:
Ah yes I see it now, thanks